lib/components/Slider/Slider.stories.tsx

```diff
@@ -50,7 +50,7 @@ export const Default: Story = {
   render,
   decorators: [
     (Story) => (
-      <div className="ds-max-w-[348px]">
+      <div className="ds-max-w-[414px]">
```

Comment:
Updated the value used in the story. 
Basically this confuses a little as the Slider will take full width by default but we are forcing the stories to have fixed width.